packages/config/config/devices/0x027a/zen22_3.7-19.0.json

```diff
@@ -449,17 +450,18 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Send each brightness level (Z-Wave disabled); Final level (Z-Wave enabled)",
+					"label": "Physical disbaled: Every brightness level; Physical enabled: Final level only",
```

Comment:
I think there was a mixup here.

Reply:
Or is this correct and the old label was incorrect?

Reply:
Their documentation says physical/zwave disabled, https://www.support.getzooz.com/kb/article/319-zen22-dimmer-switch-ver-4-01-advanced-settings/, but the only relevant thing here is if the physical control is disabled as the behavior is the same regardless if its just physical disabled or physical and zwave disabled.  You can't disable zwave control without disabling physical control.  But I do see a typo in "disbaled".  I can fix that if you are good with the current label.

Reply:
Phew... that description on their website is atrocious. I'm okay with leaving it like this (without the typo though).

Reply:
yeah, its bad.  Commit incoming for the typo.